pkg/build/api/validation/validation.go

```diff
@@ -239,7 +240,8 @@ func validateImageChange(imageChange *buildapi.ImageChangeTrigger) errs.Validati
 	if len(imageChange.Image) == 0 {
 		allErrs = append(allErrs, errs.NewFieldRequired("image"))
 	}
-	if len(imageChange.From.Name) == 0 {
+	var emptyObject kapi.ObjectReference
+	if imageChange.From == emptyObject {
```

Comment:
@bparees is there a reason why From is not (*kapi.ObjectReference) in types.go ?